myrails/configuration/dev.cfg

```diff
@@ -0,0 +1,13 @@
+[debug]
```

Comment:
if you want to add this file to gitignore as you suggested before, and to replace this file name with "`dev-example.cfg"`